blueocean-commons/src/test/java/io/jenkins/blueocean/commons/stapler/export/JSONDataWriterTest.java

```diff
@@ -56,17 +73,44 @@ public void testSimpleUsage() throws Exception {
     @Test
     public void testInheritance() throws Exception {
         assertEquals("{\"_class\":\"Container\",\"polymorph\":{\"_class\":\"Sub\",\"basic\":\"super\",\"generic\":\"sub\",\"specific\":\"sub\"}}",
-                serialize(new Container(), Container.class));
+            serialize(new Container(), Container.class));
     }
 
     public static class Sub2 extends Super {
         @Exported
-        @Override public String generic() {return "sub2";}
+        @Override
+        public String generic() {
+            return "sub2";
+        }
     }
 
     @Test
     public void testInheritance2() throws Exception { // JENKINS-13336
         assertEquals("{\"_class\":\"Sub2\",\"basic\":\"super\",\"generic\":\"sub2\"}",
-                serialize(new Sub2(), Sub2.class));
+            serialize(new Sub2(), Sub2.class));
+    }
+
+    @Test
+    public void exceptionHandling() throws IOException {
```

Comment:
stapler PR#89 only adds this test. There are some extra stuff pull in for this test, looks like they were added from other PRs. If these tests run no problem, otherwise we should pull in only what went in this PR.